### PR TITLE
Cleanup gh-pages branch

### PR DESCRIPTION
**Pull request by cbgbt**
**Issue number:**
#492

**Description of changes:**
See https://github.com/bottlerocket-os/bottlerocket-update-operator/pull/514

* Removes some leftover dotfiles
* Adds a gitignore file
* Adds license files
* Updates the README that is displayed on https://bottlerocket-os.github.io/bottlerocket-update-operator/

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,647 +1,13 @@
...
-```sh
-kubectl apply -f \
-  https://github.com/cert-manager/cert-manager/releases/download/v1.8.2/cert-manager.yaml
-```
-
-Or if you're using helm:
-
-```sh
-# Add the cert-manager helm chart
-helm repo add jetstack https://charts.jetstack.io
```
Comment (stmcginnis) [inline README.md:27]: If this is published as the helm README, do we want to keep some of these details?

Comment (cbgbt) [reply README.md:27]: I was worried about information staleness, not sure if we'll remember to update the `gh-pages` branch on release. (though I'm not opposed to adding this information in principle)

How do you feel about explicit linking to the `README` in the develop branch for installation instructions?

Comment (stmcginnis) [reply README.md:27]: Yeah, if we can't automate keeping that data up to date, linking off to the source of truth is probably the best.

**Comment by cbgbt** [comment]
Updated the README per suggestion from @stmcginnis

**Review by ecpullen — APPROVED** [review]

**Review by gthao313 — APPROVED** [review]

**Review by stmcginnis — APPROVED** [review]
Thanks!

**Pull request merged**